eldritch/monsters.py

```diff
@@ -7,7 +9,7 @@ def json_repr(self):
     return self.name
 
 
-class Monster(object):
+class Monster(Asset):
```

Comment:
I don't remember why I did this... I think it had something to do with accessing the monster's name. Regardless, it seems more consistent with the rest of the codebase.